### DISCUSSION TITLE
Possible to set timestamp when creating trace

### DISCUSSION THREAD

**Discussion by Stadly** [langfuse/langfuse#1156] [Ideas]
### Describe the feature or potential improvement

I'm moving from Promptlayer to Langfuse. I've exported my data from Promptlayer and want to import it into Langfuse. I'm using the JS/TS SDK, and things seem to work well, but I cannot set the timestamp of traces.

I can set `startTime` and `endTime` for spans and generations. Having a similar option for setting the `timestamp` of traces would be very useful when migrating data.

**Comment by marcklingen** [comment]
Agree, this is a quick improvement, will try to get to it soon. Tracking this here: https://github.com/langfuse/langfuse/issues/1157

**Reply by derlikh-smart** [reply]
Is it still possible to do?
I can't find it in Python SDK :-(
